shared/router-v2/router.desktop.js

```diff
@@ -262,11 +262,6 @@ const createElectronApp = Component => {
 const ElectronApp = createElectronApp(RootStackNavigator)
 
 const styles = Styles.styleSheetCreate({
```

Comment:
This style fixes the problem and looks like it was intended for the back button, but this `styles.back` is unused anywhere.

Reply:
rats, I tried to apply that to the back button but did it wrong, I didn't know about `WebkitAppRegion`, just `-webkit-app-region`